guide/interactions/buttons.md

```diff
@@ -81,6 +81,15 @@ client.on('interactionCreate', async interaction => {
 });
 ```
 
+::: warning
+If you're using typescript you'll need to specify the type of components your action row holds. This can be done by specifying the generic parameter in `ActionRowBuilder`.
+
+```diff
+- new ActionRowBuilder()
++ new ActionRowBuilder<ButtonBuilder>()
```

Comment:
```suggestion
+ new ActionRowBuilder<MessageActionRowComponentBuilder>()
```
So it'll accept selectmenu builders too.

Reply:
Should I add this on the select menu page as well?